src/css/base.css

```diff
@@ -0,0 +1,176 @@
+.main {
+  background: white;
+  color: #444;
+  margin: 5px 0;
+}
+
+/* .alert and .btn styles copied from Bootstrap */
+.alert {
+  padding: 1em;
+  margin-bottom: 1.5em;
+  border: 1px solid transparent;
+  border-radius: 4px;
+}
+
+.alert-danger {
+  color: #a94442;
+  background-color: #f2dede;
+  border-color: #ebccd1;
+}
+
+.alert-info {
+  color: #31708f;
+  background-color: #d9edf7;
+  border-color: #bce8f1;
+}
+
+.alert-warning {
+  color: #8a6d3b;
+  background-color: #fcf8e3;
+  border-color: #faebcc;
+}
```

Comment:
Sort of noted this in a comment above; it might be nice to pull out these colors into custom properties/variables so we can re-use them a little better.

Reply:
I pulled out these alert colors and the basic grays used by bootstrap into variables. I stayed pretty close to the naming scheme used by bootstrap for simplicity's sake (https://getbootstrap.com/docs/3.4/customize/#less-variables)

I didn't pull out all the button state colors or some other intermediary grays. I'm totally open to do more/fewer/different variable names though.

Reply:
Seems good to me; I think you did a great job splitting the difference here.